src/libraries/System.Numerics.Tensors/src/System.Numerics.Tensors.csproj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Public API not fully documented. -->
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
FYI @dotnet/area-system-numerics-tensors area owners, this assembly is missing public documentation.

Reply:
https://github.com/dotnet/runtime/issues/87710